DigitalLearningSolutions.Web/Views/LearningMenu/Section/_TutorialCard.cshtml

```diff
@@ -8,7 +8,9 @@
           @Model.TutorialName
         </h3>
         <p class="nhsuk-u-secondary-text-color">
-          @Model.TimeInformation
+          @Model.TutorialTime minutes spent
+          <span class="visually-hidden">on this tutorial</span>
+          (average tutorial time @Model.AverageTutorialTime minutes)
```

Comment:
Probably worth adding minute/minutes logic here. For that it would also probably be useful to move the logic of creating these strings back to the view model. For example having the tutorial time and average tutorial time being string fields being calculated in the view model and just slotting "on this tutorial" in the middle here would work well I think 🙂